coins/common.go

```diff
@@ -20,6 +22,13 @@ const (
 	MaxCoinPrecision = 100
 )
 
+// RelayerFeeWei and RelayerFeeETH are the fixed 0.009 ETH fee for using a swap
+// relayer to claim.
+var (
+	RelayerFeeWei = big.NewInt(9e15)
+	RelayerFeeETH = NewWeiAmount(RelayerFeeWei).AsEther()
```

Comment:
We can't have these in the `relayer` package, because we need the value in `common/types` where the offer checks are. We can't move them to the `common` package, as `common/types` depends on `common`. Given the choice of prefixing the values with `types` or `coins`, I thought `coins` was slightly better.